### PR TITLE
Add rules for 3.11 for OCP-19667 and OCP-19558

### PR DESCRIPTION
**Pull request by xiaocwan**
@yapei  This is for https://issues.redhat.com/browse/OCPQE-2287 Please review.
PASS locally http://pastebin.test.redhat.com/903844
Will add job log.

### CODE REVIEW

File: lib/rules/web/console/base/login.xyaml [hunk 1]
```diff
@@ -94,10 +94,13 @@ login_token:
...
-    selector:
-      css: "#password, #inputPassword, .login-redhat, .idp, .navigation-container"
-    timeout: 30
+  elements:
+    - selector:
+        text: Username
+      timeout: 30
+    - selector:
+        text: Password
+      timeout: 30
```
Comment (yapei) [inline lib/rules/web/console/base/login.xyaml:103]: These changes are not needed.  `css: "#password, #inputPassword, .login-redhat, .idp, .navigation-container"` are doing the same thing

**Comment by xiaocwan** [comment]
Updated 
OCP-19667: http://pastebin.test.redhat.com/904644

**Comment by xiaocwan** [comment]
Updated for OCP-19558
http://pastebin.test.redhat.com/904695

**Comment by yapei** [comment]
@liangxia  Could you help merge if no objections?

**Pull request merged**